scala/integration/intellij-bsp/src/org/jetbrains/plugins/scala/project/bsp/ScalaSdkGetter.scala

```diff
@@ -0,0 +1,33 @@
+package org.jetbrains.plugins.scala.project.bsp
+
+import com.intellij.openapi.externalSystem.service.project.IdeModifiableModelsProvider
+import org.jetbrains.magicmetamodel.impl.workspacemodel.impl.updaters.transformers.{ScalaUtil => BspScalaUtil}
+import org.jetbrains.plugins.bsp.extension.points.ScalaSdkGetterExtension
+import org.jetbrains.plugins.bsp.server.tasks.ScalaSdk
+import org.jetbrains.plugins.scala.project.ScalaLanguageLevel
+import org.jetbrains.plugins.scala.project.external.ScalaSdkUtils
+
+import java.net.URI
+import java.nio.file.Paths
+
+class ScalaSdkGetter extends ScalaSdkGetterExtension {
```

Comment:
I haven't seen the sources of BSP plugin, but the name `ScalaSdkGetterExtension` looks confusing for something which doesn't "gets scala sdk".
The logic seems to be about "registering Scala SDK" (in the module?) or "adding scala sdk to module".
Both "register" and "add" sound kinda opposite to "get"
I would suggest renaming it in the BSP plugin